c++.html.markdown

```diff
@@ -228,10 +228,20 @@ cout << myString + myOtherString; // "Hello World"
 
 cout << myString + " You"; // "Hello You"
 
+// Outputs 11 (= 5 + 6).
+// C++ string length can be found from either string::length() or string::size()
+cout << myString.length() + myOtherString.size();
+
 // C++ strings are mutable.
 myString.append(" Dog");
 cout << myString; // "Hello Dog"
 
+// C++ can handle C-style strings with related functions using cstrings
+#include <cstring>
+
+char myOldString[20] = "Hello CPP";
```

Comment:
Why 20?

Reply:
> Why 20?

No specific reason, just initialized with a sufficient size while I was choosing on string content. Shall I set it to around 9-10 for a better fit?

Reply:
10 for the null, otherwise your example won't work. Or just leave the length empty...

Reply:
> 10 for the null, otherwise your example won't work. Or just leave the length empty...

Right. Thanks for suggesting, I'll update it to 10